guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -80,6 +115,25 @@ To add details to a compute profile, complete the following steps:
 . In the *Size (GB)* field, enter the size of the storage to create on the host.
 . Click *Submit* to save the compute profile.
 
+.For CLI Users
+
+. Create a compute profile to use with the GCE compute resource:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile create --name _gce_profile_
+----
+
+. Add GCE details to the compute profile. 
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile values create --compute-profile _gce_profile_ \
+--compute-resource '_gce_cr_' \
+--volume "_size_gb=20_" \
```

Comment:
Does _size_gb with prior underscore actually appear in Doc preview. If yes, That should be fixed as size_gb. Its a size parameter and is a constant/fixed name.

Reply:
@jyejare, the underscores are used to highlight text as italics. In the built doc this line will appear in the following format:
 --volume "_size_gb=20_" \ 
^Does that look correct?

The same works for all other lines - surrounded by underscores=renders as italics.

Reply:
Make sense! Thanks for the clarification. Got to know something else.